MIGRATION.md

```diff
@@ -423,6 +424,20 @@
 
 ## From version 8.4.x to 8.5.x
 
+### Added source code panel to docs
+
+Starting in 8.5, Storybook Docs (`@storybook/addon-docs`) automatically adds a new addon panel to stories that displays a source snippet beneath each story. This works similarly to the existing [source snippet doc block](https://storybook.js.org/docs/writing-docs/doc-blocks#source), but in the story view. It is intended to replace the [Storysource addon](https://storybook.js.org/addons/@storybook/addon-storysource).
+
+If you wish to disable this panel globally, add the following line to your `.storybook/preview.js` project configuration. You can also selectively disable/enable at the story level.
+
+```js
+export default {
+  parameters: {
+    docsSourcePanel: { disable: true },
```

Comment:
Not sure I think this is the right API.

We already have `parameters.docs.source.X`: https://storybook.js.org/docs/api/doc-blocks/doc-block-source

Wouldn't it make sense to keep in line with that, and have it as `parameters.docs.source.panel = bool` instead?

Reply:
I see that this has already been discussed, before.

Personally, I highly disagree with the direction.